lib/workers/repository/updates/generate.spec.ts

```diff
@@ -1,49 +1,62 @@
 import { codeBlock } from 'common-tags';
-import { getConfig, partial } from '../../../../test/util';
-import type { RenovateConfig, UpdateType } from '../../../config/types';
+import { getConfig } from '../../../../test/util';
+import type { UpdateType } from '../../../config/types';
 import { NpmDatasource } from '../../../modules/datasource/npm';
 import type { BranchUpgradeConfig } from '../../types';
 import { generateBranchConfig } from './generate';
 
-let defaultConfig: RenovateConfig;
+const {
+  commitMessage,
+  commitMessagePrefix,
+  commitMessageAction,
+  commitMessageTopic,
+  commitMessageExtra,
+} = getConfig();
+let requiredDefaultOptions = {};
 
 beforeEach(() => {
   jest.resetAllMocks();
-  defaultConfig = getConfig();
+  requiredDefaultOptions = {
+    commitMessage,
+    commitMessagePrefix,
+    commitMessageAction,
+    commitMessageTopic,
+    commitMessageExtra,
+  };
 });
 
 describe('workers/repository/updates/generate', () => {
   describe('generateBranchConfig()', () => {
     it('does not group single upgrade', () => {
-      const branch = partial<BranchUpgradeConfig>([
+      const branch: BranchUpgradeConfig[] = [
```

Comment:
```suggestion
      const branch = [
```
i think we don't need it. the satisfies operator should do it enough